src/components/widget/RawWidget.js

```diff
@@ -128,12 +128,12 @@ export class RawWidget extends Component {
   // Datepicker is checking the cached value in datepicker component itself
   // and send a patch request only if date is changed
   handlePatch = (property, value, id, valueTo, isForce) => {
-    const { handlePatch } = this.props;
+    const { handlePatch, inProgress } = this.props;
     const willPatch = this.willPatch(property, value, valueTo);
 
     // Do patch only when value is not equal state
     // or cache is set and it is not equal value
-    if ((isForce || willPatch) && handlePatch) {
+    if ((isForce || willPatch) && handlePatch && !inProgress) {
```

Comment:
since idk how frontend is supposed to work, i'm just throwing this here: if the patch response is so slow (think `Slow 3g mode`) and the user is editing the qty 3 times in a row really fast, is it possible that only the first edit (patch) is sent to backend and the other 2 are ignored, as `inProgress` did not turn to false fast enough?

Reply:
`inProgress` is only set when you press `{enter}` to submit the quickinput. As a general rule of thumb, if you submit a form in a browser app - it's done, and you have to wait for the process to finish before doing any further changes. Until you submit it - you can change the values as many times as you want.